Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:14.04 
+RUN apt-get update
+RUN apt-get install -y curl
+RUN curl -sL https://deb.nodesource.com/setup_4.x | bash -
```

Comment:
Also you can add remote files to the host.

```dockerfile
ADD https://deb.nodesource.com/setup_4.x /tmp/install-node.sh
```

The size of that script is much smaller than all the cache (and time) that Ubuntu pulls down just to install `curl`.